components/config-provider/__tests__/__snapshots__/components.test.js.snap

```diff
@@ -10950,7 +10950,7 @@ exports[`ConfigProvider components Slider prefixCls 1`] = `
   />
   <div>
     <div
-      class="prefix-Slider-tooltip prefix-Slider-tooltip"
+      class="prefix-Slider-tooltip"
```

Comment:
Miss prefix className, should check why.

Reply:
will be resolved after updating snapshots

Reply:
not resolved 🤔

Reply:
that was a mistake, i was looking on outdated changes!